### DISCUSSION TITLE
Need help on 'Add notToBeAnInstanceOf' issue

### DISCUSSION THREAD

**Discussion by tanbirali** [robstoll/atrium#1404] [Contributor Q&A]
#1400 
I cannot understand where am I supposed to put this in the project.

**Comment by robstoll** [comment]
take a look at toBeAnInstanceOf. basically at all the same places. I can update the issue description later on if required.

**Reply by tanbirali** [reply]
It will be of great help if you will update the issue description.

**Reply by robstoll** [reply]
done. Write in this discussion in case you need further help
